src/main/java/me/totalfreedom/totalfreedommod/world/WorldRestrictions.java

```diff
@@ -27,7 +27,7 @@
 {
 
     private final List<String> BLOCKED_WORLDEDIT_COMMANDS = Arrays.asList(
```

Comment:
WTF, why is this even here? Why is this entire thing not in config...?

CC @AtlasMediaGroup/totalfreedom-developers

Reply:
good question